core/src/test/fixtures/sdk-search-fts/Item.s

```diff
@@ -2,7 +2,7 @@ CREATE TABLE item(
   id INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
```

Comment:
whats the reason for the changes in here? Would like to keep this test coverage if possible

Reply:
I think I was trying to update the test case to take advantage of more of the FTS-specific features since there's already a non-FTS version of the test that excercises the other stuff. I'm fine with reverting it though - all the changes in this PR should be covered by the new tests.

Reply:
got it, I'm fine either way. You'll want to rebase against master to fix travis (your other open PR as well too)